ui/src/components/inputs/Editor.vue

```diff
@@ -13,6 +13,14 @@
                 >
                     <unfold-more-horizontal />
                 </b-button>
+                <b-button
```

Comment:
To keep the same behaviour, I would prefer the button at the bottom bar everywhere on the Flows Page